packages/wallet-connect-sdk-react/src/index.tsx

```diff
@@ -5,43 +5,230 @@ import {ContractInvocation, ContractInvocationMulti, RpcCallResult, WcSdk} from
+     */
     accounts: string[],
     setAccounts: React.Dispatch<React.SetStateAction<string[]>>,
 
-    openPairing: () => Promise<void>,
+    /**
+     * Initialize the connection process
+     * @param topic optional pairing topic to reestablish an existing connection
+     */
     connect: (topic?: string) => Promise<void>,
+
+    /**
+     * Sends a request to the Wallet and it will call the RpcServer
+     * ```
+     * const result = await walletConnectCtx.sendRequest({
+     *   method: 'getapplicationlog',
+     *   params: ['0x7da6ae7ff9d0b7af3d32f3a2feb2aa96c2a27ef8b651f9a132cfaad6ef20724c']
+     * })
+     * ```
+     * @param request the request information object containing the rpc method name and the parameters
+     * @return the call result promise
+     */
     sendRequest: (request: RequestArguments) => Promise<RpcCallResult>,
+
+    /**
+     * Sends an 'invokefunction' request to the Wallet and it will communicate with the blockchain. It will consume gas and persist data to the blockchain.
+     * ```
+     * const senderAddress = walletConnectCtx.getAccountAddress()
+     *
+     * const from = { type: 'Address', value: senderAddress }
+     * const recipient = { type: 'Address', value: 'NbnjKGMBJzJ6j5PHeYhjJDaQ5Vy5UYu4Fv' }
+     * const value = { type: 'Integer', value: 100000000 }
+     * const args = { type: 'Array', value: [] }
+     *
+     * const resp = await walletConnectCtx.invokeFunction({
+     *    scriptHash: smartContractScripthash,
+     *    operation: 'transfer',
+     *    args: [from, recipient, value, args]
+     * })
+     * ```
+     * @param request the contract invocation options
+     * @return the call result promise. It might only contain the transactionId, another call to the blockchain might be necessary to check the result.
+     */
     invokeFunction: (request: ContractInvocation) => Promise<RpcCallResult>,
+
+    /**
+     * Sends a `testInvoke` request to the Wallet and it will communicate with the blockchain.
+     * It will not consume any gas but it will also not persist any data, this is often used to retrieve SmartContract information or check how much gas an invocation will cost.
+     * Also, the wallet might choose to not ask the user authorization for test invocations making them easy to use.
+     * ```
+     * const resp = await walletConnectCtx.testInvoke({
+     *     scriptHash: '0x010101c0775af568185025b0ce43cfaa9b990a2a',
+     *     operation: 'getStream',
+     *     args: [{ type: 'Integer', value: 17 }],
+     *     signer: { scopes: WitnessScope.None }
+     * })
+     * ```
+     * @param request the contract invocation options
+     * @return the call result promise
+     */
     testInvoke: (request: ContractInvocation) => Promise<RpcCallResult>,
+
+    /**
+     * Sends a `multiInvoke` request to the Wallet, it will concatenate all invocations and send to the blockchain all at once.
```

Comment:
```suggestion
     * Sends a `multiInvoke` request to the Wallet. It will concatenate all invocations and send them to the blockchain all at once.

```